presto-docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -23,8 +23,6 @@ Pushdown depends on a number of factors:
 * generic support for pushdown for that function in Presto
```

Comment:
what is meant here?

Reply:
that generally Presto needs to be capable of pushing down the processing of this specific function ... I was understanding that this is necessary from prior discussions. If this is not necessary or works differently we should clarify.

Reply:
Do you have an example of a function that is pushable and a function that is not?

Reply:
I dont know the details of the implementation and dont have an example. It might be wrong .. you or whoever knows the details would have to confirm

Reply:
So i do not know what is meant here, hence the question. I cannot ask a more knowledgeable person, because i dont know who would that be in this context.